src/daily_output.ts

```diff
@@ -70,30 +68,13 @@ export class DailyOutput {
 	 * - streak rewards
 	 */
 	private async finishDataAggregation(): Promise<void> {
-		// check for previous unbundled bundles
-		const prevNonUnbundledBundles: { txId: string; tip: number }[] = existsSync(NON_UNBUNDLED_BUNDLES_NAME)
-			? JSON.parse(readFileSync(NON_UNBUNDLED_BUNDLES_NAME).toString())
-			: [];
-		const unbundledTransactions = prevNonUnbundledBundles.length
-			? await getBundledTransactions(prevNonUnbundledBundles.map(({ txId }) => txId))
-			: [];
-		// track the tips of the bundles yet not unbundled at the previous run
-		prevNonUnbundledBundles.forEach(({ tip, txId }) => {
-			this.bundlesTips[txId] = tip;
+		// aggregate +1 file count to the non unbunded bundles
+		const bundleTxIDs = Object.keys(this.bundlesTips);
+		bundleTxIDs.forEach((txId) => {
+			if (!this.bundleFileCount[txId]) {
+				this.sumFile(this.bundlesTips[txId].address);
+			}
```

Comment:
Does this make the bundle at risk of being double counted when it gets unpacked?

Reply:
There's no risk: only if the bundle isn't unpacked (e.i. `this.bundleFileCount[txId] === undefined`) the file count will be bumped

Reply:
But what will happen on the next aggregation if the bundle is unpacked? Will the fileCount of that unpacked bundle carry over, and how do we track as those new file data items come in on the aggregation ?

Reply:
We don't do that anymore. If in the current run the bundle is not unpacked, then it's counted as 1